superset-frontend/src/dashboard/components/RefreshIntervalModal.jsx

```diff
@@ -74,12 +102,32 @@ class RefreshIntervalModal extends React.PureComponent {
               value={this.state.refreshFrequency}
               onChange={this.handleFrequencyChange}
             />
+            {showRefreshWarning && (
+              <div className="refresh-warning-container">
+                <Alert bsStyle="warning">
+                  <div>{refreshWarning}</div>
+                  <br />
+                  <div className="bold">
+                    {t('Are you sure you still want to proceed?')}
```

Comment:
I'd remove the `still` here because it's a bit more concise, but up to you

Reply:
removed `still`.